### PR TITLE
Blog New done Story 3

### PR DESCRIPTION
**Pull request by kentstone84**

### CODE REVIEW

File: app/views/blog/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<h1> Create a new blog </h1>
+<%= form_with model: @blog, local: true do |form| %>
```
Comment (WalkerTexas-coder) [inline app/views/blog/new.html.erb:2]: This is calling on an instance variable in the new method. add the instance variable @blog = Blog.new

File: app/views/blog/new.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <h1> Create a new blog </h1>
-<%= form_with model: @blog, local: true do |form| %>
+<%= form_with model: @blog=Blog.new do |form| %>
```
Comment (WalkerTexas-coder) [inline app/views/blog/new.html.erb]: `@blog` here

File: app/controllers/blog_controller.rb [hunk 3]
```diff
@@ -7,4 +7,7 @@ def show
     end
     def new
```
Comment (WalkerTexas-coder) [inline app/controllers/blog_controller.rb:8]: `@blog = Blog.new` here

Comment (kentstone84) [reply app/controllers/blog_controller.rb:8]: Got it. We have corrected this code in the controller and changed the view back to the original code.

**Review by WalkerTexas-coder — APPROVED** [review]

**Pull request merged**